xmtp_mls/src/client.rs

```diff
@@ -165,9 +209,58 @@ where
         Ok(installation_ids)
     }
 
+    pub(crate) async fn pull_from_topic(&self, topic: &str) -> Result<Vec<Envelope>, ClientError> {
+        let mut conn = self.store.conn()?;
+        let last_synced_timestamp_ns =
+            EncryptedMessageStore::get_last_synced_timestamp_for_topic(&mut conn, topic)?;
+
+        let envelopes = self
+            .api_client
+            .read_topic(topic, last_synced_timestamp_ns as u64)
```

Comment:
The network query API is "greater than or equals", so I think we actually want this to be `(last_synced_timestamp_ns + 1)` to avoid getting duplicate messages we will end up skipping

Reply:
Changing this, but wouldn't it make more sense to have the network query API just be "greater than"? That seems like a much cleaner system overall

Reply:
I definitely wish I had done it that way when I first wrote the query API last year. The rationale was compatibility with Waku, which worked that way. Since we are using the generic query interface, we'd have to make the change for everyone (maybe with a flag). But yes, that would be better.